modules/fts/pages/fts-architecture.adoc

```diff
@@ -0,0 +1,54 @@
+= FTS Architecture
+
+
+[abstract]
+
+
+The FTS Architecture is explained in the form of diagrams. Each diagram is co-located with its textual description.
+
+== Search Service Architecture
+
+The _Search_ Service provides extensive capabilities for natural-language querying.
+These include:
+
+* _Language-aware_ searching; allowing users to search for, say, the word `beauties`, and additionally obtain results for `beauty` and `beautiful`.
```

Comment:
`beauties` is an unusual word choice as it just isn't that commonly used.

Reply:
Let's leave it as is for now.